content/staff-wellbeing.html.erb

```diff
@@ -0,0 +1,153 @@
+              reading_time: "4 minutes", created_by: "Notre Dame High School" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Establishing a school workload group", href: "#",
+              body: "Learn how to set up a workload group.", tag: "Example", details: {
+              reading_time: "20 minutes", created_by: "Baylis Court School (Secondary)" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Prioritising wellbeing to improve staff retention", href: "#",
+              body: "Learn how prioritising wellbeing impacted staff retention.", tag: "Example", details: {
+              reading_time: "11 minutes", created_by: "Bedford Academy (Secondary)" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Prioritising wellbeing to improve staff retention", href: "#",
+              body: "Learn how prioritising wellbeing impacted staff retention.", tag: "Example", details: {
+              reading_time: "11 minutes", created_by: "Bedford Academy (Secondary)" }) %>
+
+          </ul>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container govuk-!-padding-top-6 govuk-!-padding-bottom-0">
+          <h2 class="govuk-heading-s">Take action</h2>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container">
+          <ul class="resource-card-group">
+
+            <%= render('/partials/resource_card.*', title: "21 things we did to improve staff wellbeing", href: "#",
+              body: "Read advice and tips on improving staff wellbeing.", tag: "Example", details: {
+              reading_time: "3 minutes", created_by: "King Charles I Secondary School" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Whole school approach to wellbeing", href: "#",
+              body: "Read advice and tips on improving staff wellbeing.", tag: "Example", details: {
+              reading_time: "7 minutes", created_by: "Barr Beacon School (Secondary)" }) %>
+
+          </ul>
+        </div>
+      </div>
+      <div class="govuk-grid-column-one-third govuk-!-padding-top-0">
+        <hr class="govuk-section-break--blue govuk-section-break--thick govuk-!-margin-top-6">
+
+        <div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+          <h2 class="govuk-heading-m">Promote wellbeing and flexible working in your school</h2>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+          <p class="govuk-body">
+            <a href="https://www.gov.uk/guidance/education-staff-wellbeing-charter">Education staff wellbeing charter</a>
+          </p>
+          <p class="govuk-body">
+            <a href="https://www.gov.uk/guidance/get-help-with-flexible-working-in-schools">Flexible working toolkit</a>
+          </p>
+          <p class="govuk-body">
+            <a href="#">Supporting staff mental health in schools</a>
```

Comment:
we have the link for this now:
https://improve-workload-and-wellbeing-for-school-staff.education.gov.uk/staff-wellbeing/supporting-staff-mental-health/

Reply:
Nice spot. Added.